### PR TITLE
[javascript] Add CVE-2016-10695

### PR DESCRIPTION
**Pull request by msrb**

### CODE REVIEW

File: database/javascript/2016/10695.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+cve: 2016-10695
+title: CVE in sqlite3
+description: >
+    The npm-test-sqlite3-trunk module provides asynchronous, non-blocking SQLite3 bindings. npm-test-sqlite3-trunk downloads binary resources over HTTP, which leaves it vulnerable to MITM attacks. It may be possible to cause remote code execution (RCE) by swapping out the requested resources with an attacker controlled copy if the attacker is on the network or positioned in between the user and the remote server.
+cvss_v2: 9.3
+references:
+    - https://nodesecurity.io/advisories/297
+
+affected:
+    - name: sqlite3
```
Comment (sara-02) [inline database/javascript/2016/10695.yaml]: The name should be `npm-test-sqlite3-trunk` and not just `sqlite3`.

File: database/javascript/2016/10695.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+      version:
+        - "<unable-to-determine>"
```
Comment (sara-02) [inline database/javascript/2016/10695.yaml]: https://nvd.nist.gov/vuln/detail/CVE-2016-10695?cpeVersion=2.2#vulnConfigurationsArea
`versions up to (including) 4.0.1`

File: database/javascript/2016/10695.yaml (hunk 1 above)
Comment (msrb) [reply database/javascript/2016/10695.yaml]: Good catch ;)

File: database/javascript/2016/10695.yaml (hunk 2 above)
Comment (msrb) [reply database/javascript/2016/10695.yaml]: There is no such version: https://www.npmjs.com/package/npm-test-sqlite3-trunk

**Pull request merged**